modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-api/src/main/java/com/liferay/portal/search/tuning/rankings/index/Ranking.java

```diff
@@ -3,11 +3,11 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.portal.search.tuning.rankings.web.internal.index;
+package com.liferay.portal.search.tuning.rankings.index;
```

Comment:
this is kind of related to our upgrade process for inactive -> status. remember how we needed to target classname of Ranking? well we are changing the FQCN here, so we probably need an upgrade process to change it in the json storage table (and maybe the classname_ table as well?). we might actually have to be careful about the order of the 2 upgrade processes... might be better to upgrade the classname first, and then inactive -> status